README.md

```diff
@@ -15,9 +15,11 @@ the square brackets (for multiple-choice questions).
 
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
-Your Answer:
-```text
-```
+Your Answer: when executed, your wrapper element is smaller than your content element. You can fix this by using a clearfix in CSS:
+example:
+.clearfix {
+  overflow: auto;
+}
 
```

Comment:
True, although it's important to note that the wrapper isn't necessarily "smaller" than the content element. Because the content element is floated, the wrapper does not factor the content size into its dimensions. By placing something after the floated element (in this case done by `overflow`), the container is forced to expand and accommodate the floated element.